elm-app/src/elm/Translate.elm

```diff
@@ -90,13 +90,22 @@ translate lang trans =
                         formater =
                             format "%d/%m/%Y %H:%M"
 
+                        compareFormater =
+                            format "%d/%m/%Y"
+
                         dateFormated =
                             formater date
 
+                        timeFormater =
+                            format "%H:%M"
+
                         allDatesFormated =
                             Maybe.map
                                 (\endDate ->
-                                    dateFormated ++ " - " ++ (formater endDate)
+                                    if compareFormater date == compareFormater endDate then
```

Comment:
@amitaibu I'm not sure if this is ugly, just wanted your input before I continue.
What we want here to display only the time if the event is on the same day.

Works:
![_ ___municipality](https://user-images.githubusercontent.com/7369740/27628763-b91edaba-5bf8-11e7-9a13-b93124bcdc1f.jpg)